test/cmocka/src/audio/volume/volume_process.c

```diff
@@ -76,28 +76,28 @@ static int setup(void **state)
 	list_init(&vol_state->dev->bsink_list);
 
 	/* allocate new sink buffer */
-	vol_state->sink = test_malloc(sizeof(*vol_state->sink));
+	size = parameters->frames * get_frame_bytes(parameters->sink_format, parameters->channels) *
+	       parameters->buffer_size_ms;
+
+	struct sof_ipc_buffer test_sink_desc = {
+		.size = size
+	};
+	vol_state->sink = buffer_new(&test_sink_desc);
```

Comment:
Seems that buffer_new(), follwed by frame_fmt and channels initialization is repeated many times (also in the next commit). A short helper function could reduce the code (and avoid desc allocation in the middle).

Reply:
moved util.h from mux unit tests to parent directory and used for selector and volume also